packages/flutter/test/material/text_field_test.dart

```diff
@@ -11786,3 +11825,44 @@ void main() {
     }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.iOS, TargetPlatform.macOS }));
   });
 }
+
+/// Simple test app for testing the obscure text.
+class _ObscureTextTestWidget extends StatefulWidget {
+   const _ObscureTextTestWidget({ required this.controller,
+   });
```

Comment:
either the constructor to be a single line or the parameters need to be in a separate line